website/docs/r/cse_network_block.html.markdown

```diff
@@ -0,0 +1,34 @@
+---
+layout: "sumologic"
+page_title: "SumoLogic: sumologic_network_block"
+description: |-
+  Provides a CSE Network Block
+---
+
+# network_block
+Provides a [CSE Network Block].
```

Comment:
I assume these brackets aren't needed.

Reply:
thank you @josh-williams ; done under https://github.com/SumoLogic/terraform-provider-sumologic/pull/271/commits/cac0ca4be3488a17ddb6b7c3826bff12d2c43aa6